deploy/build-k8s-infra.Jenkinsfile

```diff
@@ -154,6 +157,49 @@ pipeline {
                 }
             }
         }
+        stage('Package helm charts'){
+            steps{
+                dir ("${WORKSPACE}/deploy/helms") {
+                    script {
+                        chartNames = sh(returnStdout: true, script: 'ls').split()
+                        println (chartNames)
+                        for (chart in chartNames){
+                            sh"""
+                            sed -i 's@version: 0.1.0@version: ${Globals.buildVersion}@g' ${chart}/Chart.yaml
```

Comment:
Maybe we should use regex search pattern `^version: .*$`?

Reply:
Yes. Updated pipeline